backend/cms/views/statistics/matomo_api_manager.py

```diff
@@ -50,21 +51,6 @@ def cleanmatomo_url(self):
         elif not bool(re.match("^https://", self.matomo_url)):  # check for "https://" and set it
             self.matomo_url = self.protocol + self.matomo_url
 
-    def api_request(self, method, curl):
```

Comment:
This function is used several times in the api manager: line 80, line 90, line 101 and line 125.
If you delete it, you also have to delete the other functions (`create_instance`, `get_all_site_ids`, `get_all_sites_id_name` and `get_all_sites`). But I don't know if we need them in the future or not.

Reply:
I removed the functions. These are all legacy code, that might come in handy, if we implement a new process for the creation of an instance. But at the moment they are obsolete.